apps/sms/desktop-mock/navigator_moz_mobilemessage.js

```diff
@@ -1304,32 +1304,33 @@
   //  - id: Number specifying the message to retrieve
   //  Returns: request object
   MockNavigatormozMobileMessage.getMessage = function(id) {
-    var request = {
```

Comment:
That's because I tested forwarding a MMS in Firefox to see how this was working with my change, and there was a bug in the desktop mock -- that I fixed then.

So not directly related, but I needed this to test the change in Firefox.